HISTORY.md

```diff
@@ -1,3 +1,12 @@
+# Release 0.12.0
+
+- The interface for defining new distributions with a constrained support and making them compatible with `Turing` has changed. To make a custom distribution type `CustomDistribution` compatible with `Turing`, the user needs to define the method `bijector(d::CustomDistribution)` that returns an instance of type `Bijector` implementing the `Bijectors.Bijector` API.
+- `~` is now thread-safe when used for observations, but not assumptions yet.
+- There were some performance improvements in the automatic differentiation of functions in `DistributionsAD` and `Bijectors` leading to speeds closer to and sometimes faster than Stan's.
+- An HMC initialization bug was fixed. HMC initialization in Turing is now consistent with Stan's.
+- Sampling from the prior is now possible using `sample`.
+- Turing now uses AdvancedMH v0.5 and AbstractMCMC v1.0.
```

Comment:
Maybe it would be good to explain what is different with AbstractMCMC 1.0, namely that `psample` is removed and one instead has to specify `MCMCThreads()` or `MCMCDistributed()` as parallel sampling algorithms.

Reply:
For the HISTORY.md revisions I think it's better to just add a suggestion than to comment on the line -- everyone worked on different parts and should just directly add their updates.

Reply:
I added mine in a separate comment.